Applications/Calendar/Calendar.cpp

```diff
@@ -0,0 +1,55 @@
+#include "Calendar.h"
+#include <time.h>
+
+String Calendar::get_name_of_month(int month)
```

Comment:
This doesn't seem to logically belong to `Calendar` so let's make it a static free function.
Also, please avoid the `get_` prefix, per the project coding style.